presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -237,6 +237,7 @@
     private boolean isOptimizeMultipleApproxPercentileOnSameFieldEnabled = true;
     private boolean nativeExecutionEnabled;
     private String nativeExecutionExecutablePath = "./presto_server";
+    private Duration nativeExecutionResultFetchingRequestTimeout = new Duration(1, MINUTES);
```

Comment:
Is there any specific reasons we set 1 min as default? This should be ideally very fast as each result fetch is maxing out at 32MB in payload. The request is also not a blocking call as  it will return immediately if result is not ready.

Reply:
That's a safe number I've tried in the cluster run compared to previous default (200ms), but agreed 1 min probably is relatively high upper bound, let me changed it to 1s which is probably still working for most of the cases.

Reply:
I've accepted the PR. Please make sure to update the timeout to 10s

Reply:
Updated.